exercises/practice/pop-count/test_pop_count.c

```diff
@@ -0,0 +1,46 @@
+#include "test-framework/unity.h"
+#include "pop_count.h"
+
+void setUp(void)
+{
+}
+
+void tearDown(void)
+{
+}
+
+static void test_0_eggs(void)
+{
+   const int expected = 0;
+   TEST_ASSERT_EQUAL_INT(expected, egg_count(0));
+}
+
+static void test_1_eggs(void)
+{
+   const int expected = 1;
```

Comment:
In the tests for almost all other exercises there is a line `TEST_IGNORE();` (except in the first test), with an explaining comment in the second test.
see https://github.com/exercism/c/blob/main/exercises/practice/grains/test_grains.c#L19
That allows the tests to be enabled one by one so that the student can focus on one test at a time, similarly to Test-Driven Development.

Reply:
I was wondering about all of those `TEST_IGNORE()`, the explanation makes sense. Adapted it now.